AllASPFuncs.py

```diff
@@ -1343,4 +1341,40 @@ def contourGrid(image, numlevels = 5, subsize = 4):
         x_totalgrid.extend(xg)
         y_totalgrid.extend(yg)
 
-    return y_totalgrid, x_totalgrid
\ No newline at end of file
+    return y_totalgrid, x_totalgrid
+
+def saveLightcurves(ID, exposures, sn_path, confusion_metric, use_real_images, detim, supernova, X, use_roman, band):   
+    #First, build the lc file
+    if use_real_images:
+        identifier = str(ID)
+    else:
+        identifier = 'simulated'
+    if use_roman:
+        psftype = 'romanpsf'
+    else:
+        psftype = 'analyticpsf'
+
+
+    lc = pd.DataFrame()
+    if use_real_images:
```

Comment:
Maybe even have two `make_lightcurve` functions. 
`make_lightcurve`
`make_simulated_lightcurve`
Then you can test those separately.